frontend/apps/web/app/(mgmt)/[account]/runs/[id]/components/JobRunActivityErrors.tsx

```diff
@@ -35,6 +35,20 @@ function parseUint8ArrayToInitSchemaReport(
     return null;
   }
 }
+
+function parseUint8ArrayToReconcileSchemaReport(
+  data: Uint8Array
+): InitSchemaReport[] | null {
```

Comment:
just curious but why as an array if you are always returning a static report?

Reply:
to make it compatible with the old schema init report component